pkg/kubefed2/disable.go

```diff
@@ -65,6 +65,7 @@ type disableType struct {
 	disableTypeOptions
 }
 
+// nolint[:structcheck]
```

Comment:
is this a good approach to disable false positive, another option is to use a `golangci.yml` file to disable the structcheck in the file and `enable.go[1]`

Reply:
What is golint complaining about in this instance?

Reply:
some variables in the struct were unused. Excluded with a golangci lint file

Reply:
I'm confused.  If they were truly unused, why not just remove them?  Or if they are used, and golint isn't smart enough to figure that out, I'm not sure the check is worth enabling.

Reply:
> I have already disabled the check in this golangci file.

The struct and its vars are all private and not used directly with the form - `disableTypeOptions.xxx`, so the golint complains about that, though it was used in `disableType`. I think `disableTypeOptions` isn't useful and seems quite redundant, move all its vars and method into `disableType` still works the same. But we could keep that if you prefer that way.

Reply:
`disableTypeOptions` is intended to isolate options that will be set via flags.  Where there are values that are supplied via flags (`targetName` in this case), I am supportive of them being added to the command type (i.e. `disableType`).  Would that satisfy the golint check?